desci-server/scripts/be-node-dev.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh
 echo "Install bash and execute 'wait-for-it.sh' script"
-apt-get add bash
+apt-get install bash
 
 # Exit on error
 set -e
-./desci-server/scripts/wait-for-it.sh $PG_HOST:5432 --timeout=5 --strict -- echo "postgres up and running"
+./desci-server/scripts/wait-for-it.sh $PG_HOST:5433 --timeout=5 --strict -- echo "postgres up and running"
```

Comment:
I don't think this thing should be necessary at all if the compose service depends on postgres as `service_healthy`

Reply:
I was stuck in the same loop as sina until I switched to this.. this is allows  desci-server to access the port outside the container since it couldn't access `5432` due to the network isolation issue